flake-modules/dev/default.nix

```diff
@@ -12,9 +12,6 @@
       system,
       ...
     }:
-    let
-      fmt = pkgs.nixfmt-rfc-style;
-    in
```

Comment:
This was leftover from when we defined a nixfmt package for both `treefmt` and the pre-commit hook.